addon/mixins/model-validator.js

```diff
@@ -37,7 +37,8 @@ export default Ember.Mixin.create({
       if(Object.keys(errors).length !== 0){
         var stateToTransition = this.get('isNew') ? 'created.uncommitted' : 'updated.uncommitted';
         this.transitionTo(stateToTransition);
-        store.recordWasInvalid(this, errors);
+				var recordModel = this.adapterDidInvalidate ? this : this._internalModel;
```

Comment:
This fixes the errors with 1.13.x and allows for current uses to still work. @esbanarango